lib/routes/account.js

```diff
@@ -438,6 +438,7 @@ module.exports = (
         checkIsBlockForced()
           .then(() => customs.check(request, email, 'accountLogin'))
           .catch(extractUnblockCode)
+          .then(checkSecondaryEmail)
```

Comment:
@vladikoff Think your right, the more common case is a user attempting to login with email on accounts table, swapped them around.